### ISSUE TITLE
Cannot load external scripts

### ISSUE THREAD

**Issue by shopsoy** [devongovett/node-wkhtmltopdf#105]
Hi,

If if include the java script code in line with the html all works (the html file contains the js)
The issue is when I include the java script in a separate file and then load this in the html.
The result is that only the html is converted to pdf and the text that should be added to the htm is not converted.

Here is my server.js

var fs = require('fs');
var wkhtmltopdf = require('wkhtmltopdf');
wkhtmltopdf.command = "/opt/app/node_modules/wkhtmltox/bin/wkhtmltopdf";

wkhtmltopdf(fs.readFileSync("./reportsTemplate/test1/test1.html", "utf8"), {
	javascriptDelay: 5000,
    output: 'demo.pdf',
    pageSize: 'letter'
});

the html file test1.html and the js file filesjs.js in the zip file

Basically in the same server.js file  exists a folder reportsTemplate with the test1 folder containing the html and js files. 

[test1.zip](https://github.com/devongovett/node-wkhtmltopdf/files/1321796/test1.zip)

Can some one help me to identify the problem?
Is possible to convert html pages that uses external JavaScript files. 
Is AngularJS supported in the html?

Many tanks in advance
